routes/api/questions.js

```diff
@@ -40,9 +48,7 @@ router.get('/:id', async (req, res) => {
 router.patch('/:id', async (req, res) => {
   const record = await models.Question.findByPk(req.params.id);
   if (record) {
-    await record.update(
-      _.pick(req.body, ['prompt', 'answer_type', 'questionnaire_type', 'step'])
-    );
+    await record.update(_.pick(req.body, ['QuestionnaireId', 'prompt', 'answer_type', 'questionnaire_type', 'step']));
```

Comment:
And we also make sure we pick/set the QuestionnaireId in the create/update API calls...